### PR TITLE
Add sid to ServiceMessage schema

### PR DESCRIPTION
**Pull request by snripa**

### CODE REVIEW

**Review by artem-v — APPROVED** [review]

File: services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java [hunk 1]
```diff
@@ -93,14 +100,23 @@ public String header(String name) {
...
   public String qualifier() {
     return header(HEADER_QUALIFIER);
   }
 
+  /**
+   * Returns message's sid.
+   *
+   * @return sid string
+   */
+  public String sid() {
```
Comment (ronenhamias) [inline services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: rename to streamId

File: services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java [hunk 2]
```diff
@@ -187,6 +203,10 @@ public Builder qualifier(String qualifier) {
       return header(HEADER_QUALIFIER, qualifier);
     }
 
+    public Builder sid(String sid) {
```
Comment (ronenhamias) [inline services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: rename to streamId

Comment (snripa) [reply services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: +

File: services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java (hunk 1 above)
Comment (snripa) [reply services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: +

**Review by ronenhamias — APPROVED** [review]

**Review by artem-v — APPROVED** [review]

**Pull request merged**